about/design-goals.md

```diff
@@ -0,0 +1,29 @@
+# CloudEvents Specification Design Goals
+
+CloudEvents are typically used in a distributed system to allow for services to be loosely coupled during development, then then configured with tight-coupling for specific applications.
+
+The goal of the CloudEvents specification is to allow for different cloud providers to create interoperable event systems, that allow for third-party tools and services to provide:
+- new event sources
+- new event consumers
+- observability, such as monitoring, debugging and visualization,
+- developer tools and libraries
+
+Key properties of the system:
+- Source & action may be developed and deployed independently
+- Trigger configuration allows for actions to be invoked based on a condition (e.g. for a specific event type generated by a source, optionally filtered by specific attribute values, such as a Function which is invoked when an image is uploaded to specific S3 or Google Cloud Storage “bucket” path
+- Events are uniquely identified so that developers can write code that is idempotent
+- Common meta-data attributes allow for observability via tools that are developed without knowledge of the specific implementation of an event system
```

Comment:
I think this would also include: w/o knowledge of the specifics of the event itself - meaning the 'data'/